dhall-docs/src/Dhall/Docs.hs

```diff
@@ -94,6 +98,15 @@ parserInfoOptions =
         <>  Options.Applicative.progDesc progDesc
         )
 
+showParseError
```

Comment:
I'd maybe call this `showMarkdownParseError` or similar, to prevent confusion with Dhall code parse errors.

I assume that eventually we'll have to add some dedicated error types here, but it's probably too early right now.

Reply:
I used that function to show both dhall and markdown parse error as warnings (should rename the `Error` suffix to `Warning` I guess.

I think I'll create a dedicated type for markdown error so we don't use the same function for the both kind of parse errors